damus/Views/QRCodeView.swift

```diff
@@ -159,7 +150,10 @@ struct QRCodeView: View {
 
             Spacer()
             
-            Text("Follow me on Nostr", comment: "Text on QR code view to prompt viewer looking at screen to follow the user.")
+            
+            Text(verbatim: "Follow " +
+                            ((damus_state.pubkey.npub == pubkey.npub) ? "me" : "\(profile?.display_name ?? "")") +
+                             " on Nostr")
```

Comment:
cc @tyiu

Reply:
https://github.com/damus-io/damus/pull/2612#discussion_r1817954283

Reply:
@jb55 I have made the update as suggested by Terry